catalog/tests/dags/providers/provider_api_scripts/test_finnish_museums.py

```diff
@@ -21,12 +24,16 @@
 FROZEN_UTC_DATE = datetime.strptime(FROZEN_DATE, "%Y-%m-%d").replace(
     tzinfo=timezone.utc
 )
+
 fm = FinnishMuseumsDataIngester(date=FROZEN_DATE)
 image_store = ImageStore(provider=prov.FINNISH_DEFAULT_PROVIDER)
 fm.media_stores = {"image": image_store}
 
 
-_get_resource_json = make_resource_json_func("finnishmuseums")
+def _get_resource_json(json_name):
+    with open(os.path.join(RESOURCES, json_name)) as f:
+        resource_json = json.load(f)
+    return resource_json
```

Comment:
Same note here, we should still be using the `make_resource_json_func`.

Reply:
I found how this happened: I copied the test files from the old PR I opened in `/openverse-catalog` repo, and it didn't have the `make_resource_json_func` yet :) I reverted the changes everywhere.